sites.json

```diff
@@ -1376,6 +1376,16 @@
         ]
     },
 
+    {
+        "name": "Cybrary",
+        "url": "https://www.cybrary.it/members/steini/settings/delete-account",
```

Comment:
Hm, this URL might be problematic, since it's `/members/<your_username>/settings/delete-account`. Do you know if there's a generic URL to utilize? If not, we may need to make the URL something more generic (user profile page) and expand the notes to explain the steps.